CHANGES.md

```diff
@@ -2,6 +2,8 @@
 
 ## 9.0.8 (DR9, unreleased)
 
+- Add new i-band ``MASKBITS`` values (``SATUR_I`` and ``ALLMASK_I``)
+  ([PR#154](https://github.com/legacysurvey/legacysurvey/pull/154)).
 - Planned: actually update the raw and CP-processed data access instructions
```

Comment:
Let's move the Planned item to a 9.1.0 section. I don't see that update happening sooner than the next tag.

Reply:
Good suggestion. Addressed.